### PR TITLE
Feature/updated deployment instr

### PR DESCRIPTION
**Pull request by acies312**
updated values file and instruction for local deployment with minikube cluster

### CODE REVIEW

**Review by akuleshov7 — APPROVED** [review]

File: save-cloud-charts/save-cloud/README.md [hunk 1]
```diff
@@ -17,10 +17,17 @@ api-gateway acts as an entrypoint and svc/gateway is actually a LoadBalancer.
 
   For example, for minikube and dev profile run `kubectl --context=minikube --namespace=save-cloud create secret generic db-secrets --from_literal=spring.datasource.username=<...> <...>`
 * **save-backend** and **save-demo** expects the following secrets to be set under the secret `s3-secrets` (`kubectl create secret generic s3-secrets <...>`)
-  * `s3-storage.endpoint`
-  * `s3-storage.bucketName`
-  * `s3-storage.credentials.accessKeyId`
-  * `s3-storage.credentials.secretAccessKey`
+  * For **backend** service
+    * `backend.s3-storage.bucketName`
```
Comment (nulls) [inline save-cloud-charts/save-cloud/README.md:21]: are you sure?
we have the following properties in `save-backend/src/main/resources/application.properties`:
```
backend.s3-storage.endpoint=${s3-storage.endpoint}
backend.s3-storage.bucketName=${s3-storage.bucketName}
backend.s3-storage.prefix=cnb/files
backend.s3-storage.credentials.accessKeyId=${s3-storage.credentials.accessKeyId}
backend.s3-storage.credentials.secretAccessKey=${s3-storage.credentials.secretAccessKey}
```

So `backend.s3-storage.*` are taken from `s3-storage.*`

**Review by nulls — CHANGES_REQUESTED** [review]

File: save-cloud-charts/save-cloud/values-minikube.yaml [hunk 2]
```diff
@@ -48,3 +48,50 @@ s3:
...
+reposStorage:
+  storageClassName: csi-hostpath-sc
+  annotations:
+    everest.io/disk-volume-type: SAS
+    volume.beta.kubernetes.io/storage-provisioner: hostpath.csi.k8s.io
+
+neo4j:
+  enabled: true
+  neo4j:
+    password: DiktatTestNeo4j@123
```
Comment (kgevorkyan) [inline save-cloud-charts/save-cloud/values-minikube.yaml:68]: should we commit password?

Comment (kgevorkyan) [reply save-cloud-charts/save-cloud/values-minikube.yaml:68]: may be apply it in `save-deployment` repo?

Comment (nulls) [reply save-cloud-charts/save-cloud/values-minikube.yaml:68]: it's a test one. it's for local deployment only

File: save-cloud-charts/save-cloud/README.md (hunk 1 above)
Comment (acies312) [reply save-cloud-charts/save-cloud/README.md]: deleted unnecessary properties

**Review by nulls — APPROVED** [review]

**Pull request merged**